src/Projekt/Args.fs

```diff
@@ -1,111 +1,90 @@
 module Projekt.Args
 
 open Projekt.Types
-open System.IO
-open Nessos.UnionArgParser
 
-type private Args =
-    | Template of string
-    | [<AltCommandLine("-fxv")>] FrameworkVersion of string
-    | Organisation of string
-    | Direction of string
-    | Repeat of int
-    | Link of string
-    | Compile of bool
-with
-    interface IArgParserTemplate with
-        member s.Usage = 
-            match s with
-            | Template _ -> "init -- specify the template (library|console) [default: library]"
-            | Direction _ -> "movefile -- specify the direction (down|up)"
-            | Repeat _ -> "movefile -- specify the distance [default: 1]"
-            | FrameworkVersion _ -> "init -- specify the framework version (4.0|4.5|4.5.1) [default: 4.5]"
-            | Organisation _ -> "init -- specify the organisation"
-            | Link _ -> "addfile -- specify an optional Link attribute"
-            | Compile _ -> "addfile -- should the file be compiled or not  [default: true]"
+module Args =
+    open CommandLine
 
-let private templateArg (res : ArgParseResults<Args>) =
-    match res.TryGetResult(<@ Template @>) with
-    | Some (ToLower "console") -> Console
-    | Some (ToLower "library") -> Library
-    | None -> Library
-    | _ -> failwith "invalid template argument specified"
+    [<Verb("init", HelpText = "create a new project")>]
+    type public InitOptions = 
+        {   [<Value(0, Required = true, MetaName = "project path")>]    path : string 
```

Comment:
strictly speaking it should be a single space after { and record field names should not be lined up (personally don't care so much about the last one)

Record field names should be PascalCased